### PR TITLE
Feat/mobile emissions

### PR DESCRIPTION
**Pull request by ChewingGlass**

### CODE REVIEW

File: packages/helium-admin-cli/emissions/mobile.json [hunk 1]
```diff
@@ -3,12 +3,20 @@
     "startTime": "2023-04-18T00:00:00Z",
     "emissionsPerEpoch": "108,493,150.684932"
   },
+  {
+    "startTime": "2023-07-28T00:00:00Z",
+    "emissionsPerEpoch": "174,246,575.342466"
+  },
+  {
+    "startTime": "2023-07-29T00:00:01Z",
+    "emissionsPerEpoch": "141,369,863.013699"
```
Comment (ChewingGlass) [inline packages/helium-admin-cli/emissions/mobile.json:12]: Represents increase of 20% mapper rewards

**Review by bbalser — APPROVED** [review]

File: packages/helium-admin-cli/emissions/mobile.json [hunk 2]
```diff
@@ -3,12 +3,20 @@
     "startTime": "2023-04-18T00:00:00Z",
     "emissionsPerEpoch": "108,493,150.684932"
   },
+  {
```
[the remark this answers is not in the document]
Comment (waveform06) [reply packages/helium-admin-cli/emissions/mobile.json:6]: Checked against the emissions spreadsheet - MOBILE numbers correct

**Review by waveform06 — APPROVED** [review]
MOBILE numbers correct

**Review by madninja — APPROVED** [review]

File: packages/helium-admin-cli/emissions/mobile.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (abhay) [reply packages/helium-admin-cli/emissions/mobile.json:6]: Verified this amount is correct. 32,876,712.328767 missing in yesterday's mint.

File: packages/helium-admin-cli/emissions/mobile.json [hunk 3]
```diff
@@ -3,12 +3,20 @@
...
+    "startTime": "2023-07-28T00:00:00Z",
+    "emissionsPerEpoch": "174,246,575.342466"
```
Comment (abhay) [inline packages/helium-admin-cli/emissions/mobile.json:8]: confirmed this represents the addition of the 20% mapper rewards bucket until the halving date.

File: packages/helium-admin-cli/emissions/mobile.json (hunk 1 above)
Comment (abhay) [reply packages/helium-admin-cli/emissions/mobile.json:12]: confirmed post halving number.

File: packages/helium-admin-cli/emissions/mobile.json [hunk 4]
```diff
@@ -3,12 +3,20 @@
...
+  },
+  {
+    "startTime": "2023-07-29T00:00:01Z",
+    "emissionsPerEpoch": "141,369,863.013699"
+  },
   {
     "startTime": "2023-08-01T00:00:00Z",
-    "emissionsPerEpoch": "54,098,360.655738"
+    "emissionsPerEpoch": "70,491,803.278689"
```
Comment (abhay) [inline packages/helium-admin-cli/emissions/mobile.json:16]: confirmed post-halving number with a 366 day year.

File: packages/helium-admin-cli/emissions/mobile.json [hunk 5]
```diff
@@ -3,12 +3,20 @@
...
   },
   {
     "startTime": "2024-08-01T00:00:00Z",
-    "emissionsPerEpoch": "54,246,575.342466"
+    "emissionsPerEpoch": "70,684,931.506849"
```
Comment (abhay) [inline packages/helium-admin-cli/emissions/mobile.json:20]: confirmed post halving number with a 365 day year.

**Review by abhay — APPROVED** [review]

**Pull request merged**